<ui version="4.0" >
 <class>Calculation</class>
 <widget class="QDialog" name="Calculation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>363</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>363</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsalat - calculation options</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Calculation methods</string>
   </property>
   <widget class="QComboBox" name="calcList" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable" >
     <bool>false</bool>
    </property>
    <property name="currentIndex" >
     <number>-1</number>
    </property>
    <property name="maxVisibleItems" >
     <number>7</number>
    </property>
    <property name="maxCount" >
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Select calculation method for fajr, maghreb and isha</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>160</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Asr options</string>
   </property>
   <widget class="QRadioButton" name="shafiiButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Shafii</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="hanafiButton" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Hanafi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Select asr method</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Duhr added minutes</string>
   </property>
   <widget class="QSpinBox" name="duhrBox" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Set duhr custom time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>59</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>minutes</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>280</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Higher Latitudes option</string>
   </property>
   <widget class="QCheckBox" name="highLatBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Higher Latitudes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Set high Latitudes option</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>410</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>410</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>280</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Hijri date adjustment</string>
   </property>
   <widget class="QSpinBox" name="hijriBox" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>-2</number>
    </property>
    <property name="maximum" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Set hijri custom date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>days</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Calculations options</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>410</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
